.github/workflows/build-hotfix.yml

```diff
@@ -65,3 +66,20 @@ jobs:
           git commit -m "chore(dependencies): Autobump oss clouddriver version"
           git pull
           git push
+      # Adding OSS docker build for SNAP
+      - name: Publish Release
+        if: startsWith(github.ref, 'refs/heads/release-')
```

Comment:
All steps checking on a branch condition like this one should only run on `release-*_hotfixes` branch. Bad things can happen if this runs in other branches.

Reply:
Fixed the if statements with the recommended limiting to release hotfixes only.